tests/common/TestRuntime.cs

```diff
@@ -186,6 +186,29 @@ public static void AssertNotDevice (string message = "This test does not run on
 #endif
 	}
 
+	public static void AssertDesktop (string message = "This test only runs on Desktops (macOS or MacCatalyst).")
+	{
+#if __MACOS__ || __MACCATALYST__
+		return;
+#endif
+		NUnit.Framework.Assert.Ignore (message);
+	}
+
+	public static void AssertNotDesktop (string message = "This test does not run on Desktops (macOS or MacCatalyst).")
+	{
+#if __MACOS__ || __MACCATALYST__
+		NUnit.Framework.Assert.Ignore (message);
+#endif
+	}
+
+	public static void AssertARM64Desktop (string message = "This test only runs on an ARM64 desktops.")
```

Comment:
I think this is a more correct name:

```suggestion
	public static void AssertNotX64Desktop (string message = "This test does not run on x64 desktops.")
```

because otherwise it sounds like the method will assert on mobile platforms as well.

Reply:
Like iPad Desktop? but is an iPad really a desktop? haha anyways yeah I think I like yours better.